scss/helpers/_colored-links.scss

```diff
@@ -1,33 +1,33 @@
-// stylelint-disable function-name-case
+// stylelint-disable function-name-case, declaration-no-important
 
 // All-caps `RGBA()` function used because of this Sass bug: https://github.com/sass/node-sass/issues/2251
 @each $color, $value in ("primary": $accessible-orange, "light": $light) { // Boosted mod: restrict to primary and light
   .link-#{$color} {
-    color: RGBA(var(--#{$prefix}#{$color}-rgb, var(--#{$prefix}link-opacity, 1)));
-    text-decoration-color: RGBA(var(--#{$prefix}#{$color}-rgb), var(--#{$prefix}link-underline-opacity, 1));
+    color: RGBA(var(--#{$prefix}#{$color}-rgb), var(--#{$prefix}link-opacity, 1)) !important;
+    text-decoration-color: RGBA(var(--#{$prefix}#{$color}-rgb), var(--#{$prefix}link-underline-opacity, 1)) !important;
```

Comment:
Just wondering in this file if we shouldn't use `if($enable-important-utilities, !important, null)` instead of `!important` everywhere upstream (done in Boosted for `background-color`)

Reply:
I would personnaly do it the way you describe here since we should be able to not have any !important in the code (apart when really mandatory) Should we do it on Bootstrap side first ?